srv/components/provisioner/files/cluster_sls.template

```diff
@@ -30,6 +30,12 @@ cluster:
       {%- set srvnode = "srvnode-" + ((node_num + 1) | string) %}
       - {{ srvnode }}
     {%- endfor %}
+  durability:
+    storage-set-1:
+      sns:
+        data: {{ 4 if num_of_nodes > 1 else 1 }}
+        parity: {{ 2 if num_of_nodes > 1 else 0 }}
+        spare: 0    # earlier it was 3 incase for multinode cluster {{ 3 if num_of_nodes > 1 else 0 }} now for VM as well as for HW its 0
```

Comment:
Remove this comment
`# earlier it was 3 incase for multinode cluster {{ 3 if num_of_nodes > 1 else 0 }} now for VM as well as for HW its 0`

Reply:
Done